Library/Formula/osm2pgsql.rb

```diff
@@ -16,23 +14,42 @@ class Osm2pgsql < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "boost" => :build
```

Comment:
http://bot.brew.sh/job/Homebrew/11482/ failed which posts to https://groups.google.com/forum/#!topic/brew-test-bot/p_JPWHoLmSQ